src/app/core/common-components/entity-select/entity-select.component.html

```diff
@@ -43,12 +43,23 @@
     (optionSelected)="selectEntity($event.option.value)"
     autoActiveFirstOption
   >
-    <ng-content select="mat-option"></ng-content>
+
+    <mat-option *ngIf="filteredEntities.length < 1 && (includeInactive || getNumberOfInactive() === 0)" [disabled]="true" i18n="Message indicating that there are no matching results.">None.</mat-option>
+    <mat-option *ngIf="filteredEntities.length < 1 && !includeInactive && getNumberOfInactive() > 0" [disabled]="true" i18n="Message indicating that there are no matching results, but that a certain number of inactive entities match and pointing out the option of also showing the inactive">None, but {{ getNumberOfInactive() }} inactive {{ label }}. Also show inactive?</mat-option>
     <mat-option *ngFor="let res of filteredEntities" [value]="res">
       <app-display-entity
         [entityToDisplay]="res"
         [linkDisabled]="true"
       ></app-display-entity>
     </mat-option>
+    <mat-checkbox
+    *ngIf="getNumberOfInactive() > 0"
+    labelPosition="after"
+    (change)="toggleIncludeInactive()"
+    [checked]="includeInactive"
+    i18n="Label for checkbox|e.g. include inactive children"
+  >
+    Also show inactive {{ label }}
+  </mat-checkbox>
```

Comment:
Can we merge this and the `None, but ... inactive ...` option in to one? E.g. `Also show ... inactive`?